pvanalytics/util/_fit.py

```diff
@@ -58,8 +59,10 @@ def quartic_restricted(data, noon=720):
 
     Parameters
     ----------
-    data : Series
-        power or irradiance data.
+    x : Series
```

Comment:
```suggestion
    x : array or Series
```

Reply:
...unless you think this would require testing with array input type.

Reply:
I would feel better if we added tests with array input. Probably worth having some tests separate from the tests of orientation/tracker tests anyway.

Reply:
I think the best description for this type is numpy's 'array_like'. I'll change the type for `x`, but leave `y` as a Series.

Reply:
OK to leave as Series, since it will work with arrays but is untested.